### PR TITLE
added bamboo to the saw table

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

added bamboo to the saw table

## Proposed changes

Added bamboo blocks > bamboo plank
and
Stripped bamboo blocks > bamboo plank
in the saw mil

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.16.* - 1.20.*).
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Comment by SchnTgaiSpock** [comment]
bamboo blocks / stripped bamboo blocks are tagged in `Tag.BAMBOO_BLOCKS`

**Review by Sfiguz7 — APPROVED** [review]
Missed this completely, my bad

**Comment by J3fftw1** [comment]
This cant be merged
for some reason bukkit thinks the tag is empty

**Comment by Sfiguz7** [comment]
Bukkit what the fuck
Is it a thing that was broken in 1.20.0 and fixed in 1.20.1, has it been reported to them at all, is there something else at play...? Do we know anything basically?

**Comment by J3fftw1** [comment]
i dont see how this is broken
Bukkit litteraly gets the tag from minecraft and turns it into theirs.

also it doesnt work on what ever version...

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/TableSaw.java [hunk 1]
```diff
@@ -65,6 +66,16 @@ null, new ItemStack(Material.IRON_BLOCK), null
...
+
+        if (Slimefun.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_20)) {
+            // Bamboo blocks to Bamboo planks
+            displayedRecipes.add(new ItemStack(Material.BAMBOO_BLOCK));
+            displayedRecipes.add(new ItemStack(Material.BAMBOO_PLANKS, 4));
+
+            // Stripped Bamboo Blocks to Bamboo planks
+            displayedRecipes.add(new ItemStack(Material.STRIPPED_BAMBOO_BLOCK));
+            displayedRecipes.add(new ItemStack(Material.BAMBOO_PLANKS, 4));
+        }
```
Comment (JustAHuman-xD) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/TableSaw.java:78]: Nitpick and totally unnecessary change but maybe put this between the log to plank and plank to stick, for consistency

**Review by JustAHuman-xD — APPROVED** [review]
LGTM code wise, want to test to make sure everything is correct tho

**Comment by Sfiguz7** [comment]
Bumping this: are there new bamboo things now? Is the tag fixed now?

**Comment by J3fftw1** [comment]
nope the tag is still null

**Comment by J3fftw1** [comment]
nvm this seems to be a mockbukkit issue.
I can use the tag properly when i skip tests.
We really need to update mockbukkit
